### PR TITLE
Active edge

### PR DESCRIPTION
**Pull request by FluffMedic**
I can make branches again!
## Changelog
:cl:
fix: fix active edges in a few POIS
/:cl:

### CODE REVIEW

**Comment by ItsSelis** [comment]
There seem to be a few changes that are, most likely, not intentional? (The changelog stuff)

**Comment by FluffMedic** [comment]
I did not touch the changelog
I have no idea why that is there

File: html/changelogs_ch/archive/2024-03.yml
Comment (Raeschen) [inline html/changelogs_ch/archive/2024-03.yml:1]: Remove this file

File: html/changelogs_ch/AutoChangeLog-pr-7835.yml
Comment (Raeschen) [inline html/changelogs_ch/AutoChangeLog-pr-7835.yml:1]: Remove this file

File: html/changelogs_ch/AutoChangeLog-pr-7811.yml
Comment (Raeschen) [inline html/changelogs_ch/AutoChangeLog-pr-7811.yml:1]: remove this file

**Comment by FluffMedic** [comment]
Might be because I am trying to webedit things but trying to remove the file goes to editing

**Comment by Raeschen** [comment]
there you go

'deleting' something in a merge request means re-pasting the original version of that file into a new commit from the base of the whole PR branch itself

Alternatively you can find the commit that caused the unwanted changes, 'revert commit' on it in GK, select NO when it asks you if you immediately want to commit the changes, and then prune out/discard stuff until you have JUST the files you want to revert

**Pull request merged**